html-css/week1/index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Electrolize&family=Inter:wght@100..900&family=Montserrat:ital,wght@0,100..900;1,100..900&family=Overlock:ital,wght@0,400;0,700;0,900;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css"
+    />
+    <title>Guzide Guzelbey Esengun</title>
+  </head>
+  <body id="top">
+    <nav>
+      <input type="checkbox" id="menu-toggle" class="menu-checkbox" />
+      <label for="menu-toggle" class="hamburger-icon">&#9776;</label>
+      <ul class="navigation-list">
+        <li><a class="active" href="#hero">Home</a></li>
+        <li><a href="#about">About</a></li>
+        <li><a href="#skills">Skills</a></li>
+        <li><a href="#projects">Projects</a></li>
+        <li><a href="#hero">Contact</a></li>
+      </ul>
+    </nav>
+
+    <header id="hero">
+      <div class="hero-text">
+        <h1 class="developer-name">
+          Güzide &nbsp; GÜZELBEY &nbsp; ESENGÜN
+        </h1>
+        <h2>Front-end Developer</h2>
+      </div>
+      <div id="contact" class="hero-contact">
+        <ul class="contact-icons">
+          <li>
+            <a
+              class="github"
+              href="https://github.com/GuzideGuzelbey"
+              target="_blank"
+              ><i class="bi bi-github" style="color: #02c9c9;"></i
+            ></a>
+          </li>
+          <li>
+            <a
+              class="linkedin"
+              href="https://www.linkedin.com/in/guzide-guzelbeyesengun/"
+              target="_blank"
+              ><i class="bi bi-linkedin" style="color: #02c9c9;"></i
+            ></a>
+          </li>
+          <li>
+            <a
+              class="fcc"
+              href="https://www.freecodecamp.org/Guzide"
+              target="_blank"
+              ><img src="images/free-code-camp.png" alt="free-code-camp"
+            /></a>
+          </li>
+          <li>
+            <a class="codepen" href="https://codepen.io/guzit" target="_blank"
+              ><img src="images/codepen.png"
```

Comment:
Ensure all <img> elements have alt attributes for accessibility.

Reply:
Thanks for reminding.